src/main/java/com/alexlasota/medicalclinic/controller/PatientController.java

```diff
@@ -0,0 +1,76 @@
+package com.alexlasota.medicalclinic.controller;
+
+import com.alexlasota.medicalclinic.model.PatientDto;
+import com.alexlasota.medicalclinic.service.PatientService;
+import com.alexlasota.medicalclinic.model.Patient;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/patients")
+@RequiredArgsConstructor
+public class PatientController {
+
+    private final PatientService patientService;
+
+    @GetMapping
+    public List<PatientDto> getPatients() {
+        return patientService.getPatients()
+                .stream()
+                .map(this::convertToDto)
+                .collect(Collectors.toList());
```

Comment:
od Javy 14,15 mozemy uzywac samego toList()?

return patientService.getPatients()
                .stream()
                .map(this::convertToDto)
                .toList();